packages/dashboard/src/app/views/myStories/karma/myStories.karma.js

```diff
@@ -394,6 +394,34 @@ describe('Grid view', () => {
       );
     });
 
+    it('should filter by author', async () => {
+      // click the author toggle
+      const authorDropdown = fixture.screen.getByLabelText(
+        'Filter stories by author'
+      );
+      expect(authorDropdown).toBeTruthy();
+      await fixture.events.click(authorDropdown);
+
+      // find all author filters
+      const authorSelect = await fixture.screen.findByLabelText(
+        new RegExp(`^Option List Selector$`)
```

Comment:
I don't even know if this matters, but we probs don't need string interpolation here and can just use `/`
```suggestion
        /^Option List Selector$/
```

Reply:
yea idk why I did a regexp here. sound good